cmake-modules/AzureBuildTargetForCI.cmake

```diff
@@ -4,8 +4,21 @@
 # Defines utility functions to create build targets for CI.
 #
 
-macro(create_per_service_target_build target service)
+macro(create_per_service_target_build target service runAsSample)
```

Comment:
This is a nit, feel free to ignore it, but for most of the macros I've seen, the order of parameters puts the ServiceDirectory parameter before the executable directory - in other words, `create_per_service_target_build(service target)`, not `create_per_service_target_build(target, service)`.

Reply:
sure, we can keep consistency around it